docs/client-sdk/javascript/tutorials/filter-conversations.md

```diff
@@ -0,0 +1,129 @@
+
+* A message type
+Filter to display only conversations that are notifications or alerts, for example. Conversation IDs are different from [content types](/docs/dev-concepts/content-types). Using a custom content type does not require that you use conversation IDs.
+
+* A subject line  
+Filter to display only conversations with a specific subject line, as in email
+
+* An NFT  
+Filter to display only conversations about an NFT, though not based on ownership of an NFT
+
+<!--What might this concrete NFT use case sound like? It is not based on ownership - but on a specific NFT contract address (NFT collection) or on a contract address and token ID - a specific instance of an NFT in a collection?-->
+
+Use this tutorial to learn how to use conversation IDs and metadata to filter conversations in your app.
+
+Conversation IDs and metadata are **not required**. Using conversation IDs affects the user experience in certain apps built with XMTP, so implement them only if you have a strategic need to filter conversations.
+
+<!--Only conversation Ids create separate conversation between two addresses, correct? Providing metadata only doesn't have this effect, correct?-->
+
+For example, when you set an ID for a conversation between two users, if they are messaging using multiple apps built with XMTP, they may see multiple conversations between their addresses in apps that use a portable inbox model. To learn more about this scenario, see [Label conversations](label-conversations).
+
+<!--okay to mention portable inbox? The Label conversations tutorial talks a little bit more about what we mean by portable inbox-->
+
+
+## Set a conversation ID
+
+Set the `conversationId` when creating a conversation. `conversationId` values are private and encrypted.
+
+:::tip
+
+As a best practice, start your `conversationId` with a domain unique to your app to help avoid collisions between your app and other apps on the XMTP network.
+
+:::
+
+This example sets `mydomain.xyz/notif` as the `conversationID` when creating a conversation with recipient address `0x3F11b27F323b62B159D2642964fa27C46C841897`:
+
+```js showLineNumbers
+// Start a scoped conversation with ID mydomain.xyz/notif
+const conversation1 = await xmtp.conversations.newConversation(
+  '0x3F11b27F323b62B159D2642964fa27C46C841897',
+  {
+    conversationId: 'mydomain.xyz/notif',
+  }
+)
+```
+
+This `conversationId` indicates that these conversations are notification messages created by `mydomain.xyz`, for example. The `mydomain.xyz/notif` conversation ID can then be used to display only notification messages created by the app.
+
+
+## Set conversation metadata
+
+In addition to setting a conversation ID when creating a conversation, you can set metadata to use as an additional filter. Conversation metadata is private and encrypted.
+
+<!--Can you use metadata without a conversationId?-->
+
+<!--does this use case make sense? any ideas for a better one? =)-->
```

Comment:
@neekolas @saulmc any thoughts on this example use case below? thank you...

Reply:
I like that it's developer focused and outside the realm of the usual use cases we talk about. That said, I think something like providing a display name/nickname for the conversation might be more straightforward. We can also use the app identifier XRC as an example here once we have it.

Reply:
Thank you for this, Saul - got it. Updating now.